src/main/java/io/bitrise/trace/step/InjectTraceTask.java

```diff
@@ -254,8 +256,18 @@ static boolean updateBuildScriptContent(final String path) throws IOException {
         final String regex = "buildscript[ \\t\\n\\r]*\\{";
         final Pattern pattern = Pattern.compile(regex);
         final Matcher matcher = pattern.matcher(codeContent);
+        final List<Range> literalsPos = findStringLiterals(codeContent);
         if (matcher.find()) {
-            final String updatedContent = matcher.replaceFirst(getUpdatedBuildScriptContent());
+            boolean result = true;
```

Comment:
not sure what this `result` variable is holding? whether it's a stringLiteralResult, L265 is confusing because it says "if result is false, return false" - perhaps this variable would be better named something like `foundUpdateLocation` and should return false on L266 (not the variable) to make it clear it's not returning the variable, but that the buildscript hasn't been updated?

Reply:
Renamed it to matcherFindResult, to indicate it is the result for matcher.find(). Changed to false to make it more clear.

Reply:
thanks! that's much easier to read 😎